server/reports/stock-detail/2_3_0/src/query.graphql

```diff
@@ -2,8 +2,11 @@ query StockDetail($storeId: String, $itemCodeOrName: String) {
   stockLines(
     storeId: $storeId
     page: { first: 5000 }
-    filter: { itemCodeOrName: { like: $itemCodeOrName }, hasPacksInStore: true }
-    sort: { key: itemName, desc: false }
+    filter: {
+      itemCodeOrName: { like: $itemCodeOrName }
+      hasPacksInStore: true
+      isActive: true
```

Comment:
I got confused about this, i thought it won't work (graphql won't allow it), but it seems to work somehow ? (isActive is not part of filter input)

Reply:
Have added the filter on stock line